### DISCUSSION TITLE
Improve Media Manager (hide dot files & create folders)

### DISCUSSION THREAD

**Discussion by appinteractive** [tinacms/tinacms#3834] [General]
Hey,

Two Points:

1. The Media Manager shows dot files like `.gitkeep` or `.DS_Store` files. Dot files should not be displayed at all in the Media Manager, as there is no point in it.

<img width="379" alt="grafik" src="https://user-images.githubusercontent.com/432828/233330013-f4e75cb6-10e4-44ad-acf7-08ffdcc53300.png">

2. Please add the ability of creating Folders

**Comment by DevGlitch** [comment]
I agree the Media Manager needs some improvements in regards to the files and folders it shows.

The dot files should definitely not be shown.

Also, it would be very useful for us to be able to exclude or match specific folders that we want to show or not. This would greatly improve the Media Manager.
